test/http-basic-test.js

```diff
@@ -116,6 +116,31 @@ test('success: no response', () => {
   })
 })
 
+test('success: no response (conditional)', () => {
+  const server = http.createServer((req, res) => {
+    serverTiming({
+      enabled: req => req.url.includes('?debug=true')
```

Comment:
I know this is an example code. but this query does not match `&debug=true` .
FYI: if you would be better to check query as follows:

```javascript
enabled: req => {
  const url = new URL(req.url, req.headers.host)
  return url.searchParams("debug") === "true"
}
```

This code is correcter than previous one, but this is just test code, so Just FYI.

Reply:
Yeah, I just simplified it for the tests, but I've updated with more robust checks now :)